### PR TITLE
feat(frontend): add umami tracking code

### LINKED ISSUE #2254: 🚀 [Feature] Analytics tool to track user behaviour
## 🚀 Feature

Analytics tool to track user behaviour

- Do we need it?
- What tool we want to use?
- Implement

### PR DESCRIPTION
**Pull request by roschaefer**
Motivation
----------
This will setup umami for presenter and frontend.

How to test
-----------
1. Copy `UMAMI_WEBSITE_ID` from `frontend/.env.dist` into `frontend/.env`
2. `docker compose up`
3. Visit http://localhost:3000/
4. Watch tracked analytics at https://eu.umami.is/websites/848016ec-3c22-403c-86be-9b21864d1256

close #2254

### CODE REVIEW

File: frontend/.env.dist [hunk 1]
```diff
@@ -24,3 +24,5 @@ PUBLIC_ENV__META__DEFAULT_AUTHOR="DreamMall Verlag GbR"
 
 # SENTRY_DSN=
 # SENTRY_ENVIRONMENT=
+
+# UMAMI_WEBSITE_ID="848016ec-3c22-403c-86be-9b21864d1256"
```
Comment (mahula) [inline frontend/.env.dist:28]: Shouldn't this ID related to our Umami account be exposed here?

**Review by mahula — COMMENTED** [review]
Nice proof of work.
Only the one question.
Else, I would approve.

File: frontend/.env.dist (hunk 1 above)
Comment (roschaefer) [reply frontend/.env.dist:28]: No, this is not a secret. The tracking code including this ID is entirely visible in the app's HTML code. 

Plausible has "Shields" where you can restrict incoming traffic by IP address, Country or Hostname. Umami has nothing, as far as I know.

Comment (mahula) [reply frontend/.env.dist:28]: Good.
just wanted to be sure.

**Review by mahula — APPROVED** [review]

**Pull request merged**